src/Workspaces/Core/Portable/Shared/Extensions/TypeInfoExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Shared.Extensions
```

Comment:
Let's keep this file under src/Workspaces/Core/Portable/Utilities/NullableHelpers, so that way it's in the same folder of the rest of the stuff we're going to delete once we ratify the new compiler API. If the compiler ultimately adopts this pattern then we're just going to have something like ConvertedTypeWithBlah directly as properties, and these extensions would go away.

Reply:
I moved to NullableExtensions, seems reasonable there